style.css

```diff
@@ -258,21 +258,26 @@ p {
 }
 
 .see-project-button {
+  border-radius: 8px;
   padding: 12px;
   border: solid 1px #396df2;
-  border-radius: 8px;
   gap: 10px;
   top: 12px;
-  color: #fff;
+  color: #5e6c84;
+  font-family: poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
 }
 
-.text-see-project .see-project-button {
-  background-color: #396df2;
-  transition: background-color 120ms ease-in-out, box-shadow 120ms ease-in-out;
+.see-project-button :active {
+  color: #fff;
 }
 
 .text-see-project .see-project-button:hover {
-  box-shadow: 0 8px 16px #6070ff;
+  background-color: #6070ff;
 }
```

Comment:
- [ ] You are almost done, good job making the changes just one small bit on your hover effect please change the color property also, so that it matches the figma template like 
`.text-see-project .see-project-button:hover {
  background-color: #6070ff;
  color: #fff;
}`
you sholud add the color property on all your buttons hover effect , see your design now :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206282191-d98c56e2-0a39-4b41-b2b9-2d2b80bef046.png)

How it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206282309-ef78ee62-f1af-41d4-8183-fda00346d137.png)

Reply:
@PinkMoon25  Done